test/jasmine/tests/hover_label_test.js

```diff
@@ -2062,7 +2062,7 @@ describe('Test hover label custom styling:', function() {
             });
             assertCommonLabel({
                 path: ['rgb(255, 255, 255)', 'rgb(68, 68, 68)'],
-                text: [13, 'Arial', 'rgb(255, 255, 255)']
```

Comment:
Ha yeah, that would be a problem! FWIW, we'd have had a *chance* anyway of catching this bug when we wrote the test if it said something like
```
path: {bg: 'rgb(255, 255, 255)', border: 'rgb(68, 68, 68)'}, // or fill & line...
text: [13, 'Arial', 'rgb(255, 255, 255)']
```
So it'd be immediately obvious that the path bg matches the font color, and the font cannot be seen!
As it is `path` has two colors so it's unclear what they mean. `text` is unambiguous because all items are different types, so it's OK as an array, though still perhaps awkward as you need to know the order to *write* the test.